patterns/dxp/requests.md

```diff
@@ -0,0 +1,35 @@
+# Requests
+
+Making requests is a common task in the frontend world.
+
+In [`liferay-portal`](https://github.com/liferay/liferay-portal/), we provide the [`fetch`](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/fetch.es.js) function which is a very thin wrapper on top of the existing [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) function implemented in modern browsers. Additionally, a [polyfill](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-compatibility/frontend-compatibility-ie/build.gradle#L351) is also served for Internet Explorer 11, to ensure the `fetch` function is defined.
```

Comment:
I suggest hitting "y" on GitHub when you're grabbing links like:

https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/fetch.es.js

That turns it into an "evergreen" URL that will never break unless we rewrite history (which we won't):

https://github.com/liferay/liferay-portal/blob/73fa9b927147e5a547a2cce44dd1b46ae9ab390d/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/fetch.es.js

It's all too easy for these links to become dead and broken at the next rename, and that's worse than looking at something slightly stale, I think.

This is especially the case in your second link, which points at a specific line number: that kind of link will easily break and start pointing at the wrong line unless you grab one pointing at a specific commit by hitting "y".